### ISSUE TITLE
lib/Wrap.js does not exist

### ISSUE THREAD

**Issue by seekshiva** [casesandberg/react-bounds#3]
When I use `var bounds = require('react-boulds'); bounds.wrap(Element)`, I get this error in webpack

```
ERROR in ./~/react-bounds/lib/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./Wrap.jsx in /Users/shiva/project/node_modules/react-bounds/lib
```

I noticed the the src folder has a Wrap.jsx, which is required in index.js, but while converting it to lib, the jsx file gets renamed as .js, and that is causing the issue.

I'm monkey-patching for now by making lib/index.js to require 'Wrap.js' instead.

**Comment by ejfrancis** [comment]
+1 I'm seeing the same issue

**Comment by davincho** [comment]
What's about linking in `package.json` directly to `lib/Wrap.js`?

**Comment by tonyxiao** [comment]
Seeing the same issue as well.

**Comment by jonstuebe** [comment]
looks like this is still an issue. going to try and fix it as this would work perfectly for a project I'm working on.

**Comment by casesandberg** [comment]
Sorry about this, it has been fixed if you upgrade to `react-bounds@0.1.1`
